lib/straight/order.rb

```diff
@@ -163,6 +163,7 @@ def status_changed?
         @status_changed
       end
 
+      # @deprecated
       def tid
```

Comment:
Because this will be a breaking change. If there are merchants integrated with our API, they should have time to migrate their code.